app/views/layouts/travel_insurance_admin.html.erb

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
stray empty line

Reply:
I just copied from the rad admin layout and both of them had the empty line. Removed.